doc/index.md

```diff
@@ -0,0 +1,521 @@
+| ![](./images/dsa-icon-rotate-mode-32.png "Rotate mode toggle") | Toggle map navigation control between panning and map rotation |
+
+![](./images/dsa-v-nav-demo.gif "Navigation tools reorienting the map")<br>*Navigation tools orienting the map*
+
+The navigation toolbar can be turned off in the [Settings panel](#settings-panel).
+
+<!-- ## ![](./images/dsa-icon-menu-home-32.png) Compass -->
+### Compass
+
+There is a compass on the display's lower-right corner. Its red pointer points to north. Touching the compass will rotate the map so the top points toward north.
+
+The [compass tool] is part of ArcGIS Runtime Toolkit for Qt.
+
+### Map context menu
+
+The map's context menu appears when you long-press on the map. The menu varies depending on what is at the map location you press.
+
+| Menu item       | Operation          | Terrain | Own location | Graphic/Feature |
+| :---:           | ---                | :---:   | :---:        | :---:           |
+| `Coordinates`   | Show coordinates   | X       | X            | X               |
+| `Elevation`     | Show elevation     | X       | X            | X               |
+| `Follow`        | Follow track       |         | X            | X               |
+| `Identify`      | Identify track     | X       | X            | X               |
+| `Line of sight` | [Show my line of sight to this track](#line-of-sight) |  | X | X |
+| `Observation`   | [Create observation report](#observation-report) | X | X | X |
+| `Viewshed`      | [Show viewshed from track](#viewshed)  | X | X | X |
+
+### Tool categories
+
+Tools are categorized by workflows, shown on the toolbar. Select one of the workflows on the toolbar to select tools to be displayed on the command bar.
+
+- Vehicle app: Toolbar appears vertically down the display's left side.
+- Handheld app: Toolbar horizontally across the bottom of the display.
+
+The tool categories include:
+
+|  |  |
+| :---: | --- |
+| ![](./images/dsa-icon-dsa-32.png) | Return command bar to start |
+| ![](./images/dsa-icon-map-tools-32.png) | [Map tools](#map-tools) (manage map elements) |
+| ![](./images/dsa-icon-create-reports-32.png) | [Create report tools](#create-report-tools) (such as observation reports) |
+| ![](./images/dsa-icon-analysis-tools-32.png) | [Analysis tools](#exploratory-visual-analysis) (viewshed, line of sight) |
+|  ![](./images/dsa-icon-alert-tools-32.png)  | [Alert tools](#alerts-and-conditions) (proximity, geofence, attribute-driven) |
+| ![](./images/dsa-icon-markup-tools-32.png)  | [Markup tools](#markup-tools) (sketching) |
+
+## Settings panel
+
+![](./images/dsa-tool-settings.png "Setting panel")<br>*Settings panel (DSA Vehicle)*
+
+The settings panel accesses the app's About information, app settings, and app close (for desktop platforms).
+
+- Vehicle app: Control is at the bottom of the tools category panel on the left of the display.
```

Comment:
Line 102 refers to this as a toolbar rather than a 'tools category panel' so maybe that's more appropriate here? (You can see this "Settings" gear icon in the line 57 screenshot.) Otherwise, it looks like the term "tools" is lacking a possessive apostrophe.

```suggestion
- Vehicle app: Control is at the bottom of the toolbar on the left of the display.
```

Reply:
To me it appears that "tools" is plural, not possessive. And with that, this name feels intentional and I would rather keep as-is.